Library/Homebrew/brew.rb

```diff
@@ -8,9 +8,14 @@
 RUBY_TWO = RUBY_VERSION.split(".").first.to_i >= 2
 raise "Homebrew must be run under Ruby 2!" unless RUBY_TWO
 
+homebrew_library_path = File.dirname(File.realpath(__FILE__))
+$:.unshift(homebrew_library_path)
+$:.unshift("#{homebrew_library_path}/vendor/bundler")
+require "setup"
```

Comment:
I'd have a minor preference for this, since it'd be consistent with how non-standalone Bundler is used and I think makes it more clear what's going on:

```ruby
$:.unshift("#{homebrew_library_path}/vendor")
require "bundler/setup"
```

Alternatively, a require relative should suffice here?